lua/lazyvim/plugins/extras/lang/tex.lua

```diff
@@ -0,0 +1,61 @@
+return {
+  {
+    "folke/which-key.nvim",
+    opts = {
+      defaults = {
+        ["<LocalLeader>l"] = { name = "+vimtex" },
+      },
+    },
+  },
+
+  -- Add BibTeX/LaTeX to treesitter
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "bibtex", "latex" })
+      end
+      if type(opts.highlight.disable) == "table" then
+        vim.list_extend(opts.highlight.disable, { "latex" })
+      else
+        opts.highlight.disable = { "latex" }
+      end
+    end,
+  },
+
+  {
+    "lervag/vimtex",
+    version = "*",
+    ft = { "bib", "tex" },
+    keys = {
+      { "<LocalLeader>ld", "<Cmd>VimtexDocPackage<CR>", desc = "Open Documentation" },
+    },
+    config = function()
+      vim.opt.conceallevel = 2
+      vim.g.vimtex_complete_enabled = 0 -- use texlab for completion
+      vim.g.vimtex_mappings_disable = { ["n"] = { "K" } } -- disable `K` as the confilict to LSP hover
+      vim.g.vimtex_quickfix_method = vim.fn.executable("pplatex") == 1 and "pplatex" or "latexlog"
+
+      vim.g.vimtex_view_method = vim.fn.has("mac") == 1 and "skim" or "zathura"
```

Comment:
These are opinionated choices and might not be installed by everyone, best to leave it to the user probs

Reply:
I agree with line 37-39, but still want to keep line 36.

By default, vimtex maps `K` to `<Cmd>VimtexDocPackage<CR>` and will not override existing keymaps. Usually LSP should be initialised before vimtex, which means it's impossible to have an empty `K` binding. However, I think line 36 could be a mark for people who read vimtex's manual about this function.

Reply:
36 is totally fine, I was only talking about 39

Reply:
Already removed

Reply:
BTW, in line 37, `latexlog` is provided by vimtex.